src/test/admin.category.test.ts

```diff
@@ -0,0 +1,100 @@
+import {
+  changeCategoryStatus,
+  createCategory,
+  ListCategory,
+  searchListCategory,
+  updateCategory,
+} from '../service/admin.category.service';
+import { AppDataSource } from '../config/database';
+import { Category } from '../entities/Category';
+import { faker } from '@faker-js/faker';
+import { DEFAULT_PAGE, PAGE_SIZE } from '../utils/constants';
+const categoryRepository = AppDataSource.getRepository(Category);
+
+let connection;
+
+beforeAll(async () => {
+  connection = await AppDataSource.initialize();
+});
+
+afterAll(async () => {
+  await connection.destroy();
+});
+
+// create category
+describe('createCategory', () => {
```

Comment:
So sánh `describe` với `it`

Reply:
Describe được sử dụng để nhóm các bài kiểm tra có liên quan lại với nhau.
It được sử dụng để định nghĩa các bài kiểm tra cụ thể ạ!